CryptoAnalysis/src/test/java/tests/providerdetection/ProviderDetectionTests.java

```diff
@@ -1,8 +1,11 @@
 package tests.providerdetection;
 
 import static org.junit.Assert.assertEquals;
+
+import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore("Currently not adaptable to architecture")
```

Comment:
Can you add the reason why its not adaptable atm

Reply:
It is hard to explain, we should talk about it in the meeting. We cannot keep the provider detection as it is right know, but adapting it requires some clarifications. I disabled it because it has no impact on the analysis since it is only an option to configure the ruleset selection. Therefore, with or without the provider detection, the analysis stays the same. I could also open an issue